CHANGELOG.md

```diff
@@ -25,6 +26,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - deps: upgrade `tailing-sidecar` subchart from `v0.8.0` to `v0.9.0` [#3408]
 - chore: update OpenTelemetry Collector to v0.90.1-sumo-0 [#3439]
 - deps: update telegraf operator to `v1.3.12` [#3443]
+- feat(opentelemetry): send information about k8s version [#3463]
+- chore: update OpenTelemetry Collector to v0.92.0-sumo-0 [#3510]
```

Comment:
Could you remove `chore: update OpenTelemetry Collector to v0.90.1-sumo-0 [#3439]`?

Reply:
ok will do